src/components/TsvList/TsvList.js

```diff
@@ -127,6 +129,7 @@ TsvList.propTypes = {
   setCurrentCheck: PropTypes.func,
   setItemIndex: PropTypes.func,
   selectedQuote: PropTypes.object,
+  shouldDisableClick: PropTypes.bool,
```

Comment:
Small suggestion - maybe add a comment here about what shouldDisableClick does and why it's needed?